assets/components/paymentButtons/stripePopUpButton/stripePopUpButton.scss

```diff
@@ -1,22 +1,20 @@
 .component-stripe-pop-up-button {
-  background-color: gu-colour(neutral-1);
+  box-sizing: border-box;
+  background-color: gu-colour(news-garnett-highlight);
   border: none;
   border-radius: 600px;
-  color: #fff;
+  color: gu-colour(garnett-neutral-1);
   cursor: pointer;
   display: block;
   font-family: $gu-sans-web;
   font-size: 14px;
   height: $cta-height;
+  margin-top: $gu-v-spacing;
```

Comment:
We try to keep the positioning properties in the style sheet of the page, if not it is difficult to predict the final position of the element in the page (if we import the element in more than one page.)

Reply:
I think this is true, the margin in this case is not a property of the button itself, it's dependent on the context in which it's used.

Reply:
Ahhh, yes, that makes sense @svillafe @Ap0c . I'm going to put this back with the one off contributions page.